plugins/modules/aci_interface_policy_storm_control.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/python
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: aci_interface_policy_storm_control
+short_description: Manage Storm Control interface policies (stormctrl:IfPol)
+description:
+- Manage CDP interface policies on Cisco ACI fabrics.
+options:
+  stormctrl_policy:
+    description:
+    - The Storm Control interface policy name.
+    type: str
+    aliases: [ name ]
+  description:
+    description:
+    - The description for the Storm interface policy name.
+    type: str
+    aliases: [ descr ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+  rate:
+    description:
+    - The bandwidth rate for all packet types.
+    type: int
+  burstRate:
+    description:
+    - The bandwidth burst rate of all packet types.
+    type: int
+  bcRate:
+    description:
+    - The bandwidth rate for broadcast packets.
+    type: int
+  bcBurstRate:
+    description:
+    - The bandwidth burst rate for broadcast packets.
+    type: int
+  mcRate:
+    description:
+    - The bandwidth rate for multicast packets.
+    type: int
+  mcBurstRate:
+    description:
+    - The bandwidth burst rate for multicast packets.
+    type: int
+  uucRate:
+    description:
+    - The bandwidth rate for unknown unicast packets.
+    type: int
+  uucBurstRate:
+    description:
+    - The bandwidth burst rate for unknown unicast packets.
+    type: int
+  ratePps:
+    description:
+    - The packet per second rate for all packet types.
+    type: int
+  burstPps:
+    description:
+    - The packet per second burst rate for all packet types.
+    type: int
+  bcRatePps:
+    description:
+    - The packet per second rate for broadcast packets.
+    type: int
+  bcBurstPps:
+    description:
+    - The packet per second burst rate for broadcast packets.
+    type: int
+  mcRatePps:
+    description:
+    - The packet per second rate for multicast packets.
+    type: int
+  mcBurstPps:
+    description:
+    - The packet per second burst rate for multicast packets.
+    type: int
+  uucRatePps:
+    description:
+    - The packet per second rate for unknown unicast packets.
+    type: int
+  uucBurstPps:
+    description:
+    - The packet per second burst rate for unknown unicast packets.
+    type: int
+  isUcMcBcStormPktCfgValid:
```

Comment:
I don't think we should expose this parameter but this should be auto-calculated value in code based on input provided, because All Types will be "Invalid" and Unicast, Broadcast, Multicast will be "Valid".


<br class="Apple-interchange-newline">